test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -46,6 +46,7 @@
 
   <ItemGroup Condition=" '$(TargetFramework)' == '$(NETFXTargetFramework)' ">
     <ProjectReference Include="..\..\..\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj" />
+    <ProjectReference Include="..\..\..\src\NuGet.Clients\NuGet.PackageManagement.VisualStudio\NuGet.PackageManagement.VisualStudio.csproj" />
```

Comment:
Probably transitively included through NuGet.Commandline :)

Reply:
This is `NuGet.Clients\...`. I believe you're thinking of `NuGet.Core\...`

Reply:
I missed the .VisualStudio suffix. mb, resolving this.